padle/padle.py

```diff
@@ -496,12 +497,13 @@ async def is_day_change():
             try:
                 async with self.config.save_daily_scores() as save_daily:
                     save_daily.append([await self.config.padle_today(), await self.config.all_scores()])
-                # async with aopen("./pad-cogs/padle/monsters.txt", "r") as f:
-                #    monsters = (await f.readline()).split(",")
                 tmrw_padle = await self.config.tmrw_padle()
                 if tmrw_padle == 0:
                     MONSTERS_LIST = await self.config.monsters_list()
-                    await self.config.padle_today.set(int(random.choice(MONSTERS_LIST)))
+                    if len(MONSTERS_LIST) == 0:
+                        await self.config.padle_today.set(3260)
```

Comment:
set, as a class constant, FALLBACK_PADLE_MONSTER with a comment by its definition that this is to be used if no monster list is set

then use self.FALLBACK_PADLE_MONSTER instead of `3260`

(using 3260 here is an example of the "magic numbers in code" antipattern, you don't want "magic numbers" in your code that you don't know where they came from or why they are here, but they're doing something)